client/app/vps/import/notification.service.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+angular.module("managerApp").service("Notification", ["$q", "ovhUserPref", function ($q, ovhUserPref) {
```

Comment:
This is a new service, why is it in EcmaScript5?

Reply:
Nop it is in import, so it is imported. Anyway this will be deleted.